src/promptflow/promptflow/exceptions.py

```diff
@@ -209,3 +232,181 @@ class ValidationException(UserErrorException):
     """Exception raised when validation fails."""
 
     pass
+
+
+class ErrorInfo:
```

Comment:
Is this a public class?

Reply:
Yes

Reply:
please add docstirng for all public interfaces

Reply:
And as this class is not expected to be used by customer, we should keep it as private util.

Reply:
Ok, I understand. I will modify it to private.